README.md

```diff
@@ -24,7 +24,8 @@ Originally developed by [Carlos Tadeu Panato Junior](https://github.com/cpanato/
       - Note: Must be the team and channel handle used in the URL. For example, in the following URL, set the value to `myteam,mychannel`: https://example.com/myteam/channels/mychannel.
 
   2. Set authorized users who can accept AWS SNS subscriptions. Must be a comma-separated list of user IDs.
-      - Tip: Use the [mattermost user search](https://mattermost.com/pl/cli-mattermost-user-search) CLI command to determine a user ID.
+      - Note: This is the database ID of the user, not the username.
```

Comment:
@ewwollesen "Database ID" is not something we usually say I think. @cwarnermm Do you have a suggestion for this? We're essentially trying to say, this is the `user.Id` field instead of the `user.Username` field.

Reply:
It's kind of interesting/unique because I think this might be the only place where we use that ID instead of just the username. I agree though that calling it "Database ID" is kind of awkward. We can wait until @cwarnermm chimes in, but maybe just drop database and go for something like "This is the user ID of the user, not the username." I mainly just want to alleviate any confusion over what to enter in that field.

Reply:
> This is the user ID of the user, not the username.

Sounds good to me :+1:

Reply:
> > This is the user ID of the user, not the username.
> 
> Sounds good to me 👍

Ok I updated it, I think, but please let me know if I did it wrong.